packages/data-explorer/src/mappers/metaFilterMapper.ts

```diff
@@ -1,42 +1,77 @@
 // @ts-ignore
 import api from '@molgenis/molgenis-api-client'
 import { MetaDataApiResponse, MetaDataAttribute } from '@/types/ApiResponse'
-import { getCategoricals } from './utils'
 import { FilterDefinition } from '@/types/ApplicationState'
+import { getFieldOptions } from '@/mappers/mapperUtils'
 
-const mapMetaToFilters = async (meta: MetaDataApiResponse) => {
-  let shownFilters:string[] = []
+const MaxVisibleOptions = 10
 
-  // TODO: map all filters
-  const categoricals = await Promise.all(getCategoricals(meta.attributes).map(async (item) => {
-    const href = item && item.refEntity && item.refEntity.href
+const fieldTypeToFilterType:any = {
```

Comment:
can remove the any here, it s a map<string,string) , no typed needed here i think